File: Android/livestreamcore/src/main/java/com/trtc/uikit/livekit/livestreamcore/state/ViewState.java
package com.trtc.uikit.livekit.livestreamcore.state;

import com.trtc.uikit.livekit.livestreamcore.view.LiveStreamView;

import java.util.HashMap;
import java.util.Map;

public class ViewState {
    public LiveStreamView              localLiveView;
    public Map<String, LiveStreamView> remoteLiveViewMap = new HashMap<>();

    public void reset() {
        localLiveView = null;
        remoteLiveViewMap.clear();
    }
}

File: iOS/LiveStreamCore/Sources/State/ViewState.swift
//
//  ViewState.swift
//  LiveStreamCore
//
//  Created by jeremiawang on 2024/10/22.
//

struct ViewState {
    var localLiveView: LiveStreamView? = nil
    var remoteLiveViewMap: [String: LiveStreamView] = [:]
    var layoutMode: LayoutMode = .gridLayout
    var layoutConfig: LayoutConfig? = nil
}
